generic/tests/lvm.py

```diff
@@ -97,7 +97,8 @@ def run(test, params, env):
                 mount_lv(lv_path, session)
             utils_test.run_virt_sub_test(test, params, env, sub_type)
         elif sub_type == "lvm_clean":
-            pass
+            if not check_mount_lv(check_mount, session):
+                mount_lv(lv_path, session)
```

Comment:
I am a little bit confused why mount the lv again? I mean that we just focus on cleaning the lvm under the status of lvm is unmounting.
Hi @XueqiangWei 
Could you help to review it? Since I think you were one of the developers of this case.
Thanks.

Reply:
Sure. The original case run with lvm in a loop, if just running lvm.clean, it shoud be failed.

Reply:
> Sure. The original case run with lvm in a loop, if just running lvm.clean, it shoud be failed.

THanks for your acking.